lib/skate/settings/test_group.ex

```diff
@@ -20,6 +20,19 @@ defmodule Skate.Settings.TestGroup do
     |> convert_from_db_test_group()
   end
 
+  @spec get(integer()) :: t() | nil
+  def get(id) do
+    test_group = Skate.Repo.get(DbTestGroup, id)
+
+    if test_group do
+      test_group
+      |> Skate.Repo.preload(:users)
```

Comment:
nitpick (non-blocking): I believe `preload` works with `nil`, so this could be called immediately after `Skate.Repo.get` and won't break if there is no test group found. I think it is nice to keep the data querying function calls chained together, but that is a personal preference.

Reply:
Yep, it appears to work with `nil`! And yeah agreed it's nicer to keep them all in one pipeline.